src/lib.rs

```diff
@@ -13,8 +13,11 @@
 #![cfg_attr(feature = "clippy", allow(unreadable_literal))]
 #![cfg_attr(feature = "clippy", allow(new_without_default_derive))]
 
-// Force public structures to implement Debug
-#![deny(missing_debug_implementations)]
```

Comment:
@ebfull are you okay with this restriction being lifted?

Reply:
is it relevant for this pull request?

Reply:
This forces it for the entire crate, not just this pull request; removing it here removes it for all future PRs, which I don't think is what you intended.

Reply:
I dont recall having added it; anyways it should be fixed in the current master and in branch feature/hashing !